src/inmanta/server/services/resourceservice.py

```diff
@@ -224,6 +224,7 @@ async def get_resource(
         log_limit: int,
         connection: Optional[Connection] = None,
     ) -> Apireturn:
+        # TODO: this output will slightly change
```

Comment:
Field have been added to the DB and they leaked out here, now we remove them again, ....